README.md

```diff
@@ -1,6 +1,6 @@
 <p align="center"><a href="https://netdata.cloud"><img src="https://user-images.githubusercontent.com/1153921/95268672-a3665100-07ec-11eb-8078-db619486d6ad.png" alt="Netdata" width="300" /></a></p>
 
-<h3 align="center">The best monitoring and troubleshooting for systems, containers, applications, and infrastructure. <br />Preconfigured, opinionated, and always real-time.</h3>
+<h3 align="center">Netdata is high fidelity infrastructure monitoring and troubleshooting.<br />Open-source, free, preconfigured, opinionated, and always real-time.<br/>It is perfectly safe to install and use netdata mid-crisis, without any preparation.</h3>
```

Comment:
```suggestion
<h3 align="center">Netdata is high-fidelity infrastructure monitoring and troubleshooting.<br />Open-source, free, preconfigured, opinionated, and always real-time.<br/>It is perfectly safe to install and use netdata mid-crisis, without any preparation.</h3>
```

Reply:
@joelhans ok, let's do it like this.
Could you please do these changes yourself? It is ok to scrap my PR and create your own.